dali/python/nvidia/dali/fn.py

```diff
@@ -75,10 +75,21 @@ def op_wrapper(*inputs, **kwargs):
 
         return op_class(**init_args)(*inputs, **call_args)
 
+    def fn_wrapper(*inputs, **kwargs):
+        from nvidia.dali._debug_mode import _PipelineDebug
+        current_pipeline = _PipelineDebug.current()
+        if getattr(current_pipeline, '_debug_on', False):
+            return current_pipeline._wrap_op_call(op_wrapper, *inputs, **kwargs)
```

Comment:
Just a nitpick, maybe, it's better to pass 
```suggestion
            return current_pipeline._wrap_op_call(op_wrapper, inputs, kwargs)
```
without unpacking. I don't know if it may have any weird naming collision, and this way we just pass a list and a dict.

Reply:
done

Reply:
`_wrap_op_call` is generic like that because it is also used for arithmetic operators in `DataNodeDebug`

Reply:
done